### PR TITLE
[UID2-2892] Add new roles for okta groups

### PR DESCRIPTION
**Pull request by asloobq**

### CODE REVIEW

File: src/main/java/com/uid2/shared/auth/Role.java [hunk 1]
```diff
@@ -10,13 +10,21 @@ public enum Role {
...
     OPERATOR_MANAGER,
+    @Deprecated
     SECRET_MANAGER,
+    @Deprecated
     ADMINISTRATOR,
-    SHARING_PORTAL,
+    SHARING_PORTAL, // corresponds to custom okta scope 'uid2.admin.ss-portal'
     PRIVATE_SITE_REFRESHER, // UID2-575 - For cronjob to trigger private site data generated
-
+    ALL, // corresponds to 'developer' okta group
```
Comment (abuabraham-ttd) [inline src/main/java/com/uid2/shared/auth/Role.java:23]: ALL? Suggest a word that implies restricted operations 

DEFAULT/LIMITED access?

Comment (asloobq) [reply src/main/java/com/uid2/shared/auth/Role.java:23]: hmm..probably DEFAULT is more intuitive than ALL. I'll update this

**Review by abuabraham-ttd — APPROVED** [review]
Not sure if we merge snapshot (7.3.2-SNAPSHOT) to main branch. 

Otherwise LGTM

**Review by cody-constine-ttd — APPROVED** [review]
LGTM

**Review by Ian-Nara — APPROVED** [review]

**Comment by asloobq** [comment]
> Not sure if we merge snapshot (7.3.2-SNAPSHOT) to main branch.
> 
> Otherwise LGTM

I see commits by CI pipeline which increases the minor version and commits back https://github.com/IABTechLab/uid2-shared/pull/202

**Pull request merged**